### PR TITLE
add rest of exercise

### PR DESCRIPTION
**Pull request by JuanVimon3**

### CODE REVIEW

File: mixed-juices/package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "babel-jest": "^29.4.2",
     "core-js": "~3.27.2",
     "eslint": "^8.34.0",
+    "eslint-config-airbnb": "^19.0.4",
```
Comment (gualopezb) [inline mixed-juices/package.json:21]: It looks like you're still missing to commit the changes in the `package-lock.json` file. Please push up those updates as well.

Comment (gualopezb) [reply mixed-juices/package.json:21]: Separately, make sure this return statement is included in its own line
https://github.com/JuanVimon3/exercism-javascript/blob/juices/mixed-juices/mixed-juices.js#L61

**Review by gualopezb — CHANGES_REQUESTED** [review]

File: mixed-juices/package.json (hunk 1 above)
Comment (gualopezb) [reply mixed-juices/package.json:21]: Not a big deal, but it makes the code more readable.

**Comment by gualopezb** [comment]
@JuanVimon3 The tests execution is failing on the mixed-juices exercise.

**Review by gualopezb — APPROVED** [review]

File: mixed-juices/package.json (hunk 1 above)
Comment (JuanVimon3) [reply mixed-juices/package.json:21]: image
image

**Pull request merged**